careers/blocks/featured-card/featured-card.js

```diff
@@ -0,0 +1,23 @@
+export default function decorate(block) {
+  const blockElements = [...block.children];
+  const [backgroundImageElement, blockContent] = blockElements;
+
+  const hasBackgroundImage = backgroundImageElement.querySelectorAll('img');
+  if (hasBackgroundImage.length === 0) {
+    block.parentElement.classList.add('featured-card-wrapper-no-background');
```

Comment:
Since the parent element already has a `featured-card-wrapper` class could you just add `no-background` as an additional class. Then in CSS you could refer to it as `.featured-card-wrapper.no-background`.

Reply:
If possible, I'd like to keep it as-is for now, and review/refactor later.